### PR TITLE
Hl/79 idle external shutter

### PR DESCRIPTION
**Pull request by Neptune584**

### CODE REVIEW

File: tools/experiments/idle.py [hunk 1]
```diff
@@ -92,22 +105,28 @@ def kernel_loop(self, loop_index: TInt64):
         self.core.break_realtime()
 
         if differential_mode:
-            for kk in range(len(self.repump_aoms)):
+            for kk in range(len(self.repump_aoms_dds)):
                 # if the repump AOM is off, don't turn on and off the AOM.
```
Comment (royaready) [inline tools/experiments/idle.py:109]: If the repump AOM is off, the user mode should not be able to change to differential mode.

File: tools/experiments/idle.py [hunk 2]
```diff
@@ -92,22 +105,28 @@ def kernel_loop(self, loop_index: TInt64):
...
+                    self.repump_aoms_dds[kk].sw.on()
+            for kk in range(len(self.repump_aoms_ttl)):
+                self.repump_aoms_ttl[kk].on()
             t_count = self.pmt_counter.gate_rising_mu(interval_mu)
         else:
             t_count = self.pmt_counter.gate_rising_mu(interval_mu)
 
-        twenty_ms_mu = 20 * ms  # 20 ms time slack to prevent slowing down PMT acquisition.
+        twenty_ms_mu = (
+            20 * ms
```
Comment (royaready) [inline tools/experiments/idle.py:128]: Maybe the variable should be named `slack_time_ms`

Comment (jayich) [reply tools/experiments/idle.py:143]: delay_ or wait_time_ms?

**Review by fanmingyu212 — APPROVED** [review]

**Review by royaready — APPROVED** [review]
lgtm

**Pull request merged**